source/reference/minio-mc/mc-find.rst

```diff
@@ -106,6 +108,22 @@ Parameters
 
    Limits directory navigation to the specified depth.
 
+.. mc-cmd:: --metadata
```

Comment:
I can't flag the correct line, so commenting about this typo here.

80: `For objects on a filesyste,`